try:
    print(asda)
except Exception as e: 
    print(e)